src/meltano/core/tracking/schemas/command_event.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow event describing a Meltano command",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "command_event",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "event_uuid": {
+            "description": "A UUID that uniquely identifies an instance of this event",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+            "minLength": 36,
+            "maxLength": 36
+        },
+        "event": {
+            "description": "",
```

Comment:
Descriptions are required by SnowcatCloud for most entries. Several are blank because I have yet to fill them in, but others will probably just remain blank as they are only there to satisfy SnowcatCloud's validator.

Reply:
Thanks for calling this out. I'm totally fine with leaving these blank for now at least. We can open a new issue to fill those out after the fact, if needed.